manifest/je.dj/BoundedUIX/info.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "BoundedUIX (djsime1's fork)",
+	"id": "je.dj.BoundedUIX",
+	"description": "Makes UIX elements selectable and editable with the usual Slot gizmos and fixes UIX elements breaking when a parent is inserted. Fork of Banane9's BoundedUIX.",
+	"category": "Inspectors",
+	"sourceLocation": "https://github.com/djsime1/BoundedUIX",
+	"additionalAuthors": [
+		"Banane9"
+	],
+	"versions": {
+		"1.1.2": {
```

Comment:
Appear to have `[assembly: AssemblyProduct("GradientDriverEasing")]` and `[assembly: AssemblyTitle("GradientDriverEasing")]`